### DISCUSSION TITLE
Play CDN does not work as expected

### DISCUSSION THREAD

**Discussion by bora89** [tailwindlabs/tailwindcss#12936] [Help] [answered]
In your docs there is saying 
```
Use the Play CDN to try Tailwind right in the browser without any build step. The Play CDN is designed for development purposes only, and is not the best choice for production.
```
Where all the rest classes, like `container`, `grid` etc, it does not work at all. Your docs does not say on Pay CDN about anything else require to do. Are `container` or `grid` something special to the framework? 
If that functionality does not work - why you even suggesting it.
**What version of Tailwind CSS are you using?**

For example: https://cdn.tailwindcss.com/3.4.1

**What build tool (or framework if it abstracts the build tool) are you using?**

For example: Play CDN

**What version of Node.js are you using?**

For example: v20

**What browser are you using?**

For example: Chrome

**What operating system are you using?**

For example: macOS

**Reproduction URL**

A Tailwind Play link or public GitHub repo that includes a minimal reproduction of the bug. **Please do not link to your actual project**, what we need instead is a _minimal_ reproduction in a fresh project without any unnecessary code. This means it doesn't matter if your real project is private/confidential, since we want a link to a separate, isolated reproduction anyways.

A reproduction is **required** when filing an issue — any issue opened without a reproduction will be closed and you'll be asked to create a new issue that includes a reproduction. We're a small team and we can't keep up with the volume of issues we receive if we need to reproduce each issue from scratch ourselves.

**Describe your issue**

Describe the problem you're seeing, any important steps to reproduce and what behavior you expect instead.

**Comment by thecrypticace** [comment]
Can you provide sample code that shows how you're using the CDN?

It scans the `class` attribute of HTML tags and all supported classes (`container`, `grid`, etc… work).

**Reply by adamwathan** [reply, marked as answer]
To follow up, here's a quick demo showing it working as expected with the `grid` and `container` classes mentioned:

https://jsfiddle.net/revftjou/2/

**Reply by bora89** [reply]
My apologies, that is a user error, blindly copied html from react component with `className` instead of `class`.
